src/TimeoutMigrationTool.ASQ.IntegrationTests/TimeoutMigrationTool.ASQ.IntegrationTests.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="NUnit" Version="3.13.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.3" />
+    <PackageReference Include="Particular.CodeRules" Version="0.8.0" PrivateAssets="All" />
+    <PackageReference Include="Particular.Approvals" Version="0.2.0" />
```

Comment:
Isn't the guidance to explicitly reference the third-party SDKs in the tests instead of using transitive dependencies? 

https://github.com/Particular/PlatformArchitecture/blob/6bf2a2359a4c6588daf6ef04cc1a8cb2dce1f9a5/repo-standards/README.md#test-projects-1

You could argue it doesn't matter that much since we are not using ranges in the tool

Reply:
I'm not sure I understand. Which third party SDK is missing?